# Que17:- When is the finally block executed?

# Ans:-
'''
The finally block in Python is always executed, regardless of whether an exception occurs or not.
'''